README.md

```diff
@@ -1,2 +1,222 @@
+  - [⭐️ Show your support ](#️-show-your-support-)
+  - [🙏 Acknowledgments ](#-acknowledgments-)
+  - [📝 License ](#-license-)
+
+<!-- PROJECT DESCRIPTION -->
+
+# 📖 Microverse React capstone project - Metrics webapp Module 3 <a name="about-project"></a>
+
 This React capstone project is about building a mobile web application to check a list of metrics (numeric values) that you will create making use of React and Redux.
+
+You will select an API that provides numeric data about a topic that you like and then build the webapp around it. The webapp will have several pages:
+
+one page with a list of items that could be filtered by some parameters; in the example below, it's a list of metrics that can be filtered by the country (imagine a search field to introduce the country name like Italy, Croatia, etc.). This page should be your homepage.
+one page for the item details; in the example, the detail page for Czech Republic cities with number of views. Use 'vercel --prod' to deploy the project to production. 
+
+The Weather App is a website designed to display weather conditions for cities along with their latitude and longitude coordinates. This website is primarily built using React and Redux. Users have the capability to search for specific locations to access their respective weather conditions. Additionally, the website is optimized for mobile devices, ensuring a responsive user experience. It also provides detailed information for each city's weather conditions.
+
+## 🛠 Built With <a name="built-with"></a>
+
+### 1- Html
+### 2- CSS
+### 3- Javascript
+### 4- React
+### 5- Redux
+### 6- Jest
+
+
+
+### Tech Stack <a name="tech-stack"></a>
+
+
+<details>
+  <summary>Client</summary>
+  <ul>
+    <li><a>Html</a></li>
+    <li><a>Css</a></li>
+    <li><a>Javascript</a></li>
+    <li><a>React</a></li>
+    <li><a>Redux</a></li>
+    <li><a>Jest</a></li>
+  </ul>
+</details>
+
+
+
+<!-- Features -->
+
+### Key Features <a name="key-features"></a>
+
+- **Display Weather**
+- **Search for Locations**
+- **Mobile Responsiveness**
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+
+
+<!-- GETTING STARTED -->
+
+## 💻 Getting Started <a name="getting-started"></a>
+
+
+
+> To get a local copy up and running, follow these steps.
+
+### Prerequisites
+
+In order to run this project you need:
+
+### 1- Web Browser
+### 2- Code Editor
+### 3- Git
+### 4- Node.js
+### 5- NPM
+### 6- React
+
+
+
+### Setup
+
+Clone this repository to your desired folder:
+
+Run This Command:
+
+
+
+```sh
+  cd my-folder
+  git clone https://github.com/Abdulwhbhussain/Metrics-Webapp-Microverse.git
+```
+
+
+### Install
+
+Install this project with:
+
+Run This Command:
+
+```sh
+  cd my-project
+  npm install
+```
+
+
+### Usage
+
+To run the project, execute the following command:
+
+> npm run start 
+
+
+### Run tests
+
+> npm run test
+
+### Deployment
+
+> Here is the Link to this Live Project:
+
+> https://space-travelers-hub-three.vercel.app/
```

Comment:
- [ ] This link is for `Space Travellers` project and not the metrics app. Kindly deploy and add the appropriate link 👍

Reply:
Ok sure!